testing/kea-hook-runscript/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: Steve HOLWEG <skytep@gmail.com>
+
+pkgname=kea-hook-runscript
+pkgver=1.1.0
+pkgrel=0
+pkgdesc="This a hook for the Kea DHCP server that allows to run an external script."
+arch="all"
+url="https://github.com/zorun/kea-hook-runscript"
+license="MPL2"
+depends="kea"
+makedepends="kea-dev kea-utils boost-dev"
+install=""
+subpackages=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/zorun/$pkgname/archive/v$pkgver.tar.gz"
+
+
+build() {
+  cd "$builddir"
+  
+  make
+}
+
+check() {
+  # Library and makefile provide no tests
+  return 0
+}
```

Comment:
instead of creating a `check` function that does not nothing, please do:
````sh
options="!check" # Library and makefile provide no tests
````

and abuild will skip running the check. It will also make it possible for us to identify all APKBUILDs without tests.